connector/MongoDBAtlasODBC.pq

```diff
@@ -15,12 +15,22 @@ MongoDBAtlasODBCType = type function(
         Documentation.FieldDescription = "Database to connect to",
         Documentation.SampleValues = {"db"}
     ]),
+    optional query as (type text meta [
+        Documentation.FieldCaption = "SQL Statement",
+        Documentation.SampleValues = {"SELECT * from table"},

```

Comment:
```suggestion
        Documentation.SampleValues = {"SELECT * FROM table"},

```
[nit] we used all caps for "SELECT" so we should do the same for "FROM" (or make them consistently all lower case)

Reply:
Ah good point.  I'll commit the suggestion 👍